docs/ops/filesystems.md

```diff
@@ -102,8 +102,8 @@ fs.<scheme>.limit.input: (number, 0/-1 mean no limit)
 fs.<scheme>.limit.output: (number, 0/-1 mean no limit)
 fs.<scheme>.limit.timeout: (milliseconds, 0 means infinite)
 fs.<scheme>.limit.stream-timeout: (milliseconds, 0 means infinite)
-fs.<scheme>.limit.rateLimitingInput: (bytes/s, 0 means infinite)
-fs.<scheme>.limit.rateLimitingOutput: (bytes/s, 0 means infinite)
+fs.<scheme>.limit.input-rate-limit: (bytes/s, 0 means infinite . By default, there is no limits)
```

Comment:
`there is no limit` (limit instead of limits)

And sorry for confusion, but I just noticed that word `limit` occurs twice in property name. It would be enough to name it:
`fs.<scheme>.limit.input-rate`
and
`fs.<scheme>.limit.output-rate`
Sorry for previous comment in wrong direction!